frame/nomination-pools/src/lib.rs

```diff
@@ -2325,6 +2636,7 @@ impl<T: Config> Pallet<T> {
 			member: member_account.clone(),
 			pool_id: member.pool_id,
 			payout: pending_rewards,
+			commission: commission_current.as_ref().map(|(c, _)| *c).unwrap_or(Zero::zero()),
```

Comment:
as_ref and that `map( *c)` are by definition canceling each other out, you probably don't need both.

Reply:
Hmm, the compiler is not liking removing `as_ref()` as `commission_current` is moved. Considering `c` is the `current[0]` and not the whole `current`, I don't think there is a way around this?

Reply:
The other syntax would be:

```
*maybe_commission.as_ref().map(|(c, _)| c).unwrap_or(&Zero::zero())
```

But I think the current one is better.